ONSdigital/rm-collection-exercise-service
# add redis client

# What and why?
Add functionality to invalidate redis cache when an exercise goes live

# How to test?
Create collection, when it goes live should be able to refresh page and it will appear in todo list. The default expiry is 10 minutes so you may want to deploy https://github.com/ONSdigital/ras-frontstage/pull/886 giving you 30minutes to create and go live exercise. Can test without this code using that 30minutes if you want to confirm that it doesn't invalidate cache. 

# Trello
https://trello.com/c/nKZNVODc/2440-ability-to-invalidate-collection-exercise-redis-cache-from-rops
